crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -544,6 +544,37 @@ fn handle_own_line_comment_between_statements<'a>(
         return CommentPlacement::Default(comment);
     }
 
+    if comment.line_position().is_end_of_line() {
+        return CommentPlacement::Default(comment);
+    }
+
+    // Comments after docstrings need a newline between the docstring and the comment, so we attach
+    // it as leading on the first statement after the docstring.
+    // (https://github.com/astral-sh/ruff/issues/7948)
+    // ```python
+    // class ModuleBrowser:
+    //     """Browse module classes and functions in IDLE."""
+    //     # ^ Insert a newline above here
+    //
+    //     def __init__(self, master, path, *, _htest=False, _utest=False):
+    //         pass
+    // ```
```

Comment:
What if there's no statement in the body after the comment? Like:

```
class ModuleBrowser:
    """Browse...."""
    # Insert a newline above
```

Reply:
Looks like it doesn't support that case yet. How else could we approach it? 🤔

Reply:
Fixed, manually inserting a newline seems to be the best option